### ISSUE TITLE
Avoid overlapping between microlitter size-ranges (H03)

### ISSUE THREAD

**Issue by mmolinajack** [nvs-vocabs/EMODnetChemVocabs#22]
Following the size ranges proposed at the TGML microlitter workshop, we would propose to review the whole vocabulary to improve the coherence of each class limits (i.e. 20-49; 50-99; 100-299 instead of 20-50; 50-100; 100-300)

**Comment by gwemon** [comment]
@mmolinajack  could you give me the full list of size classes you want please? Am I correct to remember that we decided to deprecate existing and replaced by the new ones? Currently we have the following size-classes:
H0300012	0.5 to 1.5 millimetres	0.5-1.5mm
H0300006	1 to 2 millimetres	1-2mm
H0300008	1 to 5 millimetres	1-5mm
H0300013	1.5 to 5 millimetres	1.5-5mm
H0300011	100 to 500 microns	100-500um
H0300015	15 to 50 millimetres	15-50mm
H0300007	2 to 5 millimetres	2-5mm
H0300002	20 to 200 microns	20-200um
H0300004	200 to 300 microns	200-300um
H0300005	300 microns to 1 millimetres	300um-1mm
H0300010	300 microns to 5 millimetres	300um-5mm
H0300014	5 to 15 millimetres	5-15mm
H0300009	5 to 25 millimetres	5-25mm
H0300001	less than 20 microns	<20um
H0300003	less than 200 microns	<200um

**Comment by mmolinajack** [comment]
Yes @gwemon we decided to deprecate the existing terms and create new ones. Our proposal is the following:
H0300001 less than 20 microns
H0300002 20-199 microns
H0300003 less than 200 microns
H0300004 200-299 microns
H0300005 300-999 microns
H0300006 1000-1999 microns
H0300007 2000-4999 microns
H0300008 1000-4999 microns
H0300009 5 to 25 millimetres
H0300010 300-4999 microns
H0300011 100-499 microns
H0300012 500-1499 microns
H0300013 1500-4999 microns
H0300014 5-14,999 millimetres
H0300015 15 to 50 millimetres

**Comment by gwemon** [comment]
@mmolinajack  Thank you. Please note that deprecation means that these identifiers will no longer be valid so the identifiers will all change except for <20 and <200um terms which will remain valid. Also shall I make the definitions clearer so that it clearly say that these are strict lesser than and exclude particles with size of greater than 19.5um or 199.5um respectively? Have there been an agreement on this? Also, I have used "micrometres" instead of "microns" for the new size-classes in order to be consistent with official terminology. I will do the same for the existing and replacement ones. 

However before I make all these changes (specially the deprecations), can I please double-check with you consistency across monitoring programs again. I checked the protocols in place for defining size range for sediment grain size (e.g. https://en.wikipedia.org/wiki/Grain_size) and the size-categories here follow the same pattern we had i.e. the upper limit of a size class is the lower limit of the next one. This is just one use case of size-classes of course and it could be justifiably different for marine micro-litter. So while scanning the net for some written protocols on micro-litter size measurements I found this [TGML document](https://mcc.jrc.ec.europa.eu/documents/201702074014.pdf). I see that they were, at the time (2017), recommending these size-classes: "Microplastics should be categorised according to size with a minimum level of
resolution being to allocate the material found in to size bins of approximately 100 µm (20-100 µm, 101-200 µm, 201- 300µm etc)". So have these recommendation now changed and been superseded with the new size-classes mentioned above? Is there already or will there be a protocol written that refers to how to use these new size-classes? If this is the case then would it not be preferable to wait until a common protocol is consolidated before making the changes? 

Also I realise that H03 is defined as a vocabulary for micro-litter size categories. Is that still the case? or is there an intention to also use it for beach and floating litter size categories? 

Finally, I think we should say what is meant by size? Is it, as mentioned in the same TGML report, "The size of an object as defined by its largest dimension, width or length."? This was for floating/beach litter but does it also apply to micro-litter? or does it depends on the method used for measuring the micro-litter particles?

**Comment by mmolinajack** [comment]
@gwemon thanks for the work. We will try to reply to all the points.
> @mmolinajack Thank you. Please note that deprecation means that these identifiers will no longer be valid so the identifiers will all change except for <20 and <200um terms which will remain valid. Also shall I make the definitions clearer so that it clearly say that these are strict lesser than and exclude particles with size of greater than 19.5um or 199.5um respectively? Have there been an agreement on this? Also, I have used "micrometres" instead of "microns" for the new size-classes in order to be consistent with official terminology. I will do the same for the existing and replacement ones.
>Reply: Thank you. Yes, we know that deprecation means that the terms won't be available anymore. 
If you think that it is better to make definitions clearer, indicating that the terms are strict lesser and exclude particles with size of greater than 19.5um or 199.5um respectively, for us it is ok.
We agree with micrometres.
> However before I make all these changes (specially the deprecations), can I please double-check with you consistency across monitoring programs again. I checked the protocols in place for defining size range for sediment grain size (e.g. https://en.wikipedia.org/wiki/Grain_size) and the size-categories here follow the same pattern we had i.e. the upper limit of a size class is the lower limit of the next one. This is just one use case of size-classes of course and it could be justifiably different for marine micro-litter. So while scanning the net for some written protocols on micro-litter size measurements I found this [TGML document](https://mcc.jrc.ec.europa.eu/documents/201702074014.pdf). I see that they were, at the time (2017), recommending these size-classes: "Microplastics should be categorised according to size with a minimum level of resolution being to allocate the material found in to size bins of approximately 100 µm (20-100 µm, 101-200 µm, 201- 300µm etc)". So have these recommendation now changed and been superseded with the new size-classes mentioned above? Is there already or will there be a protocol written that refers to how to use these new size-classes? If this is the case then would it not be preferable to wait until a common protocol is consolidated before making the changes?
> Reply: We see your point. Making a small review this morning, we found out that only CEFAS 2017 proposes a similar classification  (20-99 µm, 100-299 µm, 300- 999µm etc). We will write to the TGML panel of experts that were present in Ranco meeting where this was decided to make a double check with them. We will put you in cc.
> Also I realise that H03 is defined as a vocabulary for micro-litter size categories. Is that still the case? or is there an intention to also use it for beach and floating litter size categories?
> Reply: This is an important point. Micro-litter is defined up to 5mm. Meso-litter is considered up to 25 mm. In our vocabulary there are terms that are not microlitter. In Ranco's meeting the experts started to discuss the possibility of monitoring meso-litter but nothing solid was concluded. Last week, when reviewing size ranges, we decided to maintain all the size classes and keep an eye on how things evolve to evaluate later if we have to deprecate those larger size classes or if a change in the name of the vocabulary is needed (i.e. micro and meso litter size categories). What do you think?
> Finally, I think we should say what is meant by size? Is it, as mentioned in the same TGML report, "The size of an object as defined by its largest dimension, width or length."? This was for floating/beach litter but does it also apply to micro-litter? or does it depends on the method used for measuring the micro-litter particles?
>Reply: In Ranco report (which is not public but we can share it with you when the final version is available) size was defined as the length in the longest dimension.
